Document OpenDID SIOPv2 Usage (#316)
KILTprotocol/docs: docs/develop/08_opendid/04_integrate_opendid.md

    state=nitctpl7nmqcpvob7xthrw&
    token_type=bearer
```

## Self-Issued OpenID Provider v2 (SIOPv2)

> OpenDID can be configured to be compatible with [SIOPv2](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html).

Reviewer: ```suggestion
You can configure OpenDID to be compatible with [SIOPv2](https://openid.net/specs/openid-connect-self-issued-v2-1_0.html).
```

> In this case, the authorization can be done only using a DID without the need of any credentials.

Reviewer: ```suggestion
In this case, you only need a DID for the authorization, and no credentials.
```

> In order to configure the OpenDID service to allow SIOPv2, it must have a client with an empty requirements

Reviewer: ```suggestion
To configure the OpenDID service to allow SIOPv2, it must have a `client` key with an empty requirements
```

value in the `config.yaml` file.

> Initiating the SIOPv2 flow is done the same way as the [Implicit Flow](#implicit-flow) with the exception of the `nonce`

Reviewer: ```suggestion
Initiate the SIOPv2 flow the same way as the [Implicit Flow](#implicit-flow) with the exception that the `nonce`
```

> value being required.

Reviewer: ```suggestion
value is required.
```
